src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarnings/InputSuppressWarningsPatternVariables.java

```diff
@@ -0,0 +1,35 @@
+/*
+SuppressWarnings
+format = ^unchecked$*|^unused$*|.*
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, ANNOTATION_FIELD_DEF, \
+         ENUM_CONSTANT_DEF, PARAMETER_DEF, VARIABLE_DEF, METHOD_DEF, CTOR_DEF, \
+         COMPACT_CTOR_DEF, RECORD_DEF
```

Comment:
I don't see `PATTERN_VARIABLE_DEF` listed here.

Also, what about `PATTERN_DEF`? Should we be concerned about it too? A separate issue?

Reply:
> I don't see PATTERN_VARIABLE_DEF listed here.

Added. but actually [it was implicitly here because we have `(default)`](https://github.com/checkstyle/checkstyle/blob/1f939bc290344df1d3c9f5ebc5731f04fc750942/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L511-L517)

> what about PATTERN_DEF? Should we be concerned about it too?

Nope. we don't need this the annotation is on `PATTERN_VARIABLE_DEF`